File: prog/python/calculator.py
num1=int(input("Enter first Number:"))
op=input("Enter Operation:")
num2=int(input("Enter Second Number:"))
if op=='+':
    result=num1+num2
elif op=='-':
    result=num1-num2
elif op=='*':
    result=num1*num2
elif op=='/':
    result=num1/num2
elif op=='%':
    result=num1%num2
else:
    result="invalid operation"
print(result)

File: prog/c/calculator.c
#include<stdio.h>
int main()
{
	int num1,num2,output;
	char operator;
	printf("Enter a binary expression:");
	scanf("%d %c %d",&num1,&operator,&num2);
	switch(operator)
	{
		case '+':
			output=num1+num2;
			break;
		case '-':
			output=num1-num2;
			break;
		case '*':
			output=num1*num2;
			break;
		case '/':
			output=num1/num2;
			break;
		case '%':
			output=num1%num2;
			break;
		default :
			printf("invalid operation");
			break;
	}
	printf("%d",output);
	return 0;
}